python/ray/autoscaler/aws/example.yaml

```diff
@@ -77,7 +83,7 @@ setup_commands:
     # Note: if you're developing Ray, you probably want to create an AMI that
     # has your Ray repo pre-cloned. Then, you can replace the pip installs
     # below with a git checkout <your_sha> (and possibly a recompile).
-    - pip install -U ray==0.3.1
+    - pip install -U https://s3.us-east-2.amazonaws.com/richardresults/ray-0.3.1-cp35-cp35m-manylinux1_x86_64.whl || pip install -U https://s3.us-east-2.amazonaws.com/richardresults/ray-0.3.1-cp36-cp36m-manylinux1_x86_64.whl
```

Comment:
revert these?

Reply:
so merging this without new wheels will cause the autoscaler to fail (because the schema has changed with the addition of docker, which is not allowed in ray 0.3.1)